app/javascript/packages/one-time-code-input/index.js

```diff
@@ -35,6 +35,7 @@ class OneTimeCodeInput {
       });
 
       input.value = code;
+      input.dispatchEvent(new CustomEvent('input', { bubbles: true }));
```

Comment:
I just noticed in other JavaScript that `CustomEvent` is [not supported in Internet Explorer](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent#browser_compatibility). We have a polyfill though, which we can load before initializing the `OneTimeCodeInput` in the pack below.

This should look something like...

```js
import { loadPolyfills } from '@18f/identity-polyfill';

// ...

loadPolyfills(['custom-event']).then(() => new OneTimeCodeInput(fakeField).bind());
```